lib/project_types/script/layers/application/create_script.rb

```diff
@@ -36,10 +37,10 @@ def install_dependencies(ctx, language, script_name, project, project_creator)
             ProjectDependencies.install(ctx: ctx, task_runner: task_runner)
           end
 
-          def bootstrap(ctx, project_creator)
+          def bootstrap(ctx, project, project_creator)
             UI::StrictSpinner.spin(ctx.message('script.create.creating')) do |spinner|
               project_creator.bootstrap
-              spinner.update_title(ctx.message('script.create.created'))
+              spinner.update_title(ctx.message('script.create.created', project.source_path))
```

Comment:
Nitpick: it seems that we only need to pass in `project.source_path` into this method, and not the entire `project`.  It's better to design the API to tell what it needs to know.  Now that I am looking at line 34, it feels a bit inconsistent, as `project.script_name` is the `script_name`, so really, you could only pass in `project`.  Alternatively, you can pass in `project.script_name` and `project.source_path` is probably better.